### ISSUE TITLE
Put hbase-site.xml in spark conf

### ISSUE THREAD

**Issue by lucasbak** [ryba-io/ryba#102]
when running hbase jobs in spark, the best practice is to put hbase-site.xml in /etc/spark/conf, ryba should put it like hive-site, when the service is installed on the cluster

**Comment by IDerr** [comment]
I agree

**Comment by wdavidw** [comment]
ok, but if possible, only on the migration branch to facilitate rebasing

**Comment by lucasbak** [comment]
yes, I just open it to remember to push it when migration is finished

**Comment by wdavidw** [comment]
You can do it now, spark client and hbase are ready
